Create invited-expert.md (#1224)
w3c/wot: policies/invited-expert.md

## Invited Expert Policy for the W3C Web of Things (WoT) Working Group

### W3C Policy

This document extends and clarifies the W3C Invited Expert Policy at https://www.w3.org/invited-experts/ for use by the WoT Working Group and WoT Interest Group.

Some items from the W3C document are repeated for convenience.

### By Invitation

Invited Experts join the Working Group by invitation from the Working Group Chairs and Team Contact. 

Proposals, requests, and suggestions may be made to the Chairs for Invited Experts, and the Chairs may consider extending invitations. 

> If a proposal is submitted to the Chairs for consideration, the Chairs will discuss with the Team Cointact and decide whether to extend an invitation.

Reviewer: ```suggestion
If a proposal is submitted to the Chairs for consideration, the Chairs will discuss with the Team Contact and decide whether to extend an invitation.
```

### Requirements

To be considered as an Invited Expert in the WoT Working Group, an invitee must make a commitment to contribute to the Web of Things specifications and must demonstrate the ability to contribute to the specifications. 

> An Invitee must prepare and present a brief - 10 minutes - summary presentation of their background and experience, particularly in standards organizations, ans explain what/how they plan to contribute to the Web of Things specification.

Reviewer 2: ```suggestion
An Invitee must prepare and present a brief - 10 minutes - summary presentation of their background and experience, particularly in standards organizations, and explain what/how they plan to contribute to the Web of Things specification.
```

> An Invited Expert must be a contributing member of the group and is expected to regularly join WoT meetings, perform administrative duties, make and review Merge Requests, etc.

Reviewer 2: ```suggestion
An Invited Expert must be a contributing member of the group and is expected to regularly join WoT meetings, perform administrative duties, make and review Pull Requests, etc.
```

### Process

> THe Chairs and Team contact may from time to time extend an invitation to someone to join the WOT Working Group or Interest Group as an Invited Expert. Often there will be a suggestion or request sent to the Chairs and Team Contact to join the Web of Things as an Invited Expert.

Reviewer 2: ```suggestion
The Chairs and Team contact may from time to time extend an invitation to someone to join the WoT Working Group or Interest Group as an Invited Expert. Often, there will be a suggestion or request sent to the Chairs and Team Contact to join the Web of Things as an Invited Expert.
```

Consideration will be given to invitees during the weekly WoT Chairs call. The Chairs will discuss each potential invitee and typically decide whether to extend an invitation within 2 or 3 weeks.

> On completing the decision process, the Chairs will notify the potential invitee or sponsors in a timely manner.

Reviewer 2: sponsor is not previously defined

Author: Removed the reference to sponsors: 
...Chairs will notify the potential invitee in a timely manner.

> On completing the decision process, the Chairs will notify the potential invitee or sponsors in a timely manner.

Author: ```suggestion
On completing the decision process, the Chairs will notify the potential invitee in a timely manner.
```

> If a preliminary invitation is extended, the invitee will be notified in writing and arrangements will be made to schedule the presentation to the group, preferebly at an upcoming WG/IG call.

Reviewer: ```suggestion
If a preliminary invitation is extended, the invitee will be notified in writing and arrangements will be made to schedule the presentation to the group, preferably at an upcoming WG/IG call.
```

> If for some reason there is a failure to make the required arrangements, the invitation may be withdrawn and the invitee will be notified in writing. Potential invitees may need to re-apply if substantial time passes and there is still interest in becoming an IE.

Reviewer 2: ```suggestion
If for some reason there is a failure to make the required arrangements, the invitation may be withdrawn and the invitee will be notified in writing. Potential invitees may need to re-apply if substantial time passes and there is still interest in becoming an Invited Expert.
```

> After the presentation, the Chairs and Team Contact will discuss with the group at large and decide whether to proceerd wit hte invitation. If the invitation is to be extended, the invitee will be notified and will be asked to fill out the application form and proceed wit hthe organizational onboarding. If the invitee is rejected, they will be notified in a timely manner and provided a brief explanation in writing. 

Reviewer 2: ```suggestion
After the presentation, the Chairs and Team Contact will discuss with the group at large and decide whether to proceed with the invitation. If the invitation is to be extended, the invitee will be notified and will be asked to fill out the application form and proceed with the organizational onboarding. If the invitee is rejected, they will be notified in a timely manner and provided a brief explanation in writing. 
```

## Invited Expert Policy for the W3C Web of Things (WoT) Working Group

### W3C Policy

This document extends and clarifies the W3C Invited Expert Policy at https://www.w3.org/invited-experts/ for use by the WoT Working Group and WoT Interest Group.

Some items from the W3C document are repeated for convenience.

### By Invitation

Invited Experts join the Working Group by invitation from the Working Group Chairs and Team Contact. 

Proposals, requests, and suggestions may be made to the Chairs for Invited Experts, and the Chairs may consider extending invitations. 

If a proposal is submitted to the Chairs for consideration, the Chairs will discuss with the Team Contact and decide whether to extend an invitation.

### Requirements

> To be considered as an Invited Expert in the WoT Working Group, an invitee must make a commitment to contribute to the Web of Things specifications and must demonstrate the ability to contribute to the specifications. 

Reviewer 3: don't like this one, I think it is not as clear, so not merging.

> To be considered as an Invited Expert in the WoT Working Group, an invitee must make a commitment to contribute to the Web of Things specifications and must demonstrate the ability to contribute to the specifications. 

Reviewer 4: Though I'm not sure how they are to make this demonstration (regardless of the wording)...
```suggestion
To be considered as an Invited Expert in the WoT Working Group, an invitee must make a commitment to contribute to the Web of Things specifications and must demonstrate the ability to do so. 
```

An Invitee must prepare and present a brief (10 minutes or so) summary presentation of their background and experience, particularly in standards organizations, and explain what and how they plan to contribute to the Web of Things specification.

An Invited Expert must be a contributing member of the group and is expected to regularly join WoT meetings, perform administrative duties, make and review Pull Requests, etc.

### Process

The Chairs and Team Contact may from time to time extend an invitation to someone to join the WoT Working Group or Interest Group as an Invited Expert. Such invitation will often be based on a suggestion or request sent to the Chairs and/or Team Contact.

> Consideration of invitees will be made during the weekly WoT Chairs call. The Chairs will discuss each potential invitee and typically decide whether to extend an invitation within 2 or 3 weeks.

Reviewer 3: ```suggestion
Consideration of invitees will be made during the weekly WoT Chairs call. The Chairs will discuss each potential invitee and typically decide whether to extend a tentative invitation to present to the group within 2 or 3 weeks.
```

If for some reason there is a failure to make the required arrangements, the invitation may be withdrawn and the invitee will be notified in writing. Potential invitees may need to re-apply if substantial time passes and there is still interest in becoming an Invited Expert.

After the presentation, the Chairs and Team Contact will discuss with the group at large and decide whether to proceed with the invitation. If the invitation is to be extended, the invitee will be notified and will be asked to fill out the application form and proceed with the organizational onboarding. If the invitee is rejected, they will be notified in a timely manner and provided a brief explanation in writing. 

### Unsolicited Applications

> The Chairs may, at their discretion, consider unsolicited applications but are under no obligation to accept them and may simply reject them, in particular if they do not respond to requests to follow the above process.

Reviewer 4: ```suggestion
The Chairs may, at their discretion, consider unsolicited applications, but are under no obligation to accept them and may simply reject them, in particular if the applicants do not respond to requests to follow the above process.
```
